src/components/calendar/calendar.jsx

```diff
@@ -113,7 +228,7 @@ function Calendar(props) {
           onClick={onNextMonthPress}
         />
       </div>
-      <table className={styles['calendar-grid']}>
+      <table lassName={styles['calendar-grid']} role="grid" onKeyDown={onKeyDown} onFocus={onFocus} >
```

Comment:
```suggestion
      <table className={styles['calendar-grid']} role="grid" onKeyDown={onKeyDown} onFocus={onFocus} >
```

Reply:
Do we even need styles? How did this break locally but wasn't obviously broken? It might indicate that the `className` prop is unnecessary.